apps/menu-bar/src/popover/Core.tsx

```diff
@@ -129,7 +135,33 @@ function Core(props: Props) {
     [emulatorWithoutAudio]
   );
 
-  // @TODO: create another hook
+  const getDeviceByPlatform = useCallback(
+    (platform: 'android' | 'ios') => {
+      const selectedDevicesId = selectedDevicesIds[platform];
+      if (selectedDevicesId && devicesPerPlatform[platform].devices.has(selectedDevicesId)) {
+        return devicesPerPlatform[platform].devices.get(selectedDevicesId);
+      }
+
+      const devices = devicesPerPlatform[platform].devices.values();
```

Comment:
```suggestion
      const devices = devicesPerPlatform[platform]
      const selectedDevicesId = selectedDevicesIds[platform];
      if (selectedDevicesId && devices.has(selectedDevicesId)) {
        return devices.get(selectedDevicesId);
      }
```
We access `devicesPerPlatform[platform]` a few times, maybe make it variable?